Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -1022,14 +1022,8 @@ public bool Resizable
 
                 resizable = value;
 
-                if ( resizable )
-                {
-                    ExecuteAfterRender( () => InitResizable().AsTask() );
-                }
-                else
-                {
+                if ( !resizable )

```

Comment:
Why did you removed InitResizable? Wouldn't that make if not work if Resizable is changed in runtime?

Reply:
Ignore, I debugged it and noticed `await RecalculateResize();` in `OnAfterRenderAsync`.